packages/aws-rfdk/lib/core/lib/x509-certificate.ts

```diff
@@ -230,6 +274,93 @@ abstract class X509CertificateBase extends Construct {
         StringEquals: tagCondition,
       },
     }));
+
+    this.lambdaMonitor = this.addLambdaMonitorFunction({
+      uniqueValue,
+      lambdaLayers: [openSslLayer],
+      lambdaCode: props.lambdaCode,
+      tagCondition,
+      lambdaMonitorHandler: props.lambdaMonitorHandler,
+    });
+    this.addExpDateValidation({
+      uniqueValue,
+      alarmEmailAddress: props.alarmEmailAddress,
+    });
+  }
```

Comment:
I tested this with an [`X509CertificatePkcs12`](https://docs.aws.amazon.com/rfdk/api/latest/docs/aws-rfdk.X509CertificatePkcs12.html) which derives from this base class, and found that it does not emit any metric. I'm not certain if this is because:
*   PKCS#12 certs don't have expiration
*   The command you are using to extract the certificate expiry does not work with PKCS#12 certificates
*   Something else...

That being said, based on the construct's API, PKCS#12 are always converted from another `X509CertificatePem` which will have its own metric/alarm. Depending on how RFDK instructs rotating these certs, we can probably do with out metrics/alarms on `X509CertificatePkcs12`. Should we move this logic into `X509CertificatePem` instead?